lib/krikri/mapper.rb

```diff
@@ -62,13 +62,23 @@ def map(name, records)
         begin
           Registry.get(name).process_record(rec)
         rescue => e
-          Rails.logger.error("Error processing mapping for #{rec.rdf_subject}" \
-                             "\n#{e.message}\n#{e.backtrace}")
+          desc = mapping_exception_desc(rec)
+          bt = e.backtrace.join("\n")
+          Rails.logger.error("Error processing mapping.\n" \
+                             "#{desc}\n#{e.message}\n#{bt}")
           nil
         end
       end
     end
 
+    def mapping_exception_desc(rec)
```

Comment:
I'd suggest a `MappingError` as a subclass of `StandardError` that we could throw, instead.  It could capture this logic.